check_availability/main.py

```diff
@@ -33,12 +33,14 @@ def main():
   args: Dict[str, str] = docopt(__doc__)
   service: str = args['<service>']
   name: str = args['<name>']
+  # Verbosity to ERROR as default
+  verbosity:int = int(args['--verbose']) or 1
```

Comment:
No need to declare this, as you've updated the default value by setting `[default: 1]` in the docstring.

Docopt parses this and sets defaults accordingly.

Reply:
amended as per review.